t/mojo/reactor_ev.t

```diff
@@ -8,307 +8,375 @@ plan skip_all => 'EV 4.32+ required for this test!'                  unless eval
 use IO::Socket::INET;
 use Mojo::Util qw(steady_time);
 
+sub make_reactor {
+  return Mojo::Reactor::EV->new;
+}
+
+sub make_listen {
+  return IO::Socket::INET->new(Listen => 5, LocalAddr => '127.0.0.1');
+}
+
+sub make_server {
+  my $listen = shift or die 'No listen passed.';
+  my $server = $listen->accept;
+}
+
+sub get_port_listen {
+  my $listen = shift or die 'No listen passed.';
+  return $listen->sockport;
+}
+
 # Instantiation
 use_ok 'Mojo::Reactor::EV';
-my $reactor = Mojo::Reactor::EV->new;
-is ref $reactor,               'Mojo::Reactor::EV',   'right object';
-is ref Mojo::Reactor::EV->new, 'Mojo::Reactor::Poll', 'right object';
-undef $reactor;
-is ref Mojo::Reactor::EV->new, 'Mojo::Reactor::EV', 'right object';
-use_ok 'Mojo::IOLoop';
-$reactor = Mojo::IOLoop->singleton->reactor;
-is ref $reactor, 'Mojo::Reactor::EV', 'right object';
+
+subtest 'correct_types' => sub {
```

Comment:
Test descriptions are inconsistent with other tests.

Reply:
I think it is now fixed, I copied the format for subtests from other tests.